TestRail_Navigation_C+.md

```diff
@@ -0,0 +1,30 @@
+As a C+ member, you will have view-only access to Expensify's TestRail account in order to complete this task.     
+
+## Navigating TestRail
+- Expensify's TestRail account is broken down into four projects, for the purpose of this task, we will use the project titled, `NewDot - New Expensify (New Format)`. Click on this project. 
+- Within the project, you will find six headers that showcase the sub-categories of this project. You will click on `Test Suites and Cases`.
+- You will then be showed the various Test Suites and Cases available, click on `New Expensify`.
+- You now will have the overview of our various test cases that our outsourced QA team, Applause, run every day. On the right hand side, you will see a list of folders that detail different scenarios we house tests under. Clicking on one of these folders will then bring you to the list of specific test cases we currently have for the scenario in question. 
+- Clicking on one of the test cases will then take you to the overview of the steps taken by Applause to ensure everything works as expected. You will notice the expected behaviors are identified by the action word, `Verify`. 
+
+## Process for TestRail
+- Before proposing regression test steps, the C+ should first head to TestRail and identify which available scenario the bug in question best belongs in (e.g. A bug identified in the flow for updating preferred pronouns would fall under Account Settings). 
+- Following, the C+ will then determine if there's an existing test case under the scenario that we can update to encompass the new steps that will be proposed OR if we need to create a new test case to accommodate the proposed steps.
+- Once determined, the C+ will post a comment in the original GH issue which scenario the bug belongs under, why they think it belongs there, and if the test steps can fall under a current test case or if a new test case needs to be created. Please provide your reasoning for your decision in the comment and tag the BZ member to gut-check.
+  - If the BZ member agrees with the C+'s recommendation, they'll give a thumbs up on the comment. If not, a discussion will be held on where they think it might fit better, etc.
+  - There's a chance we will agree to not update/create a test case for the bug in question, depending on the bug. 
+- Once we've come to agreement where the test will live, the C+ will tag the Contributor whose PR solved the bug in a comment giving them the green light to propose test steps.
+  - If we're updating a current test case, the C+ will post what the current steps are in the GH for the Contributor to find where they can best insert their proposed test steps.
```

Comment:
Hmm this one is worded a bit wonky i think - the point is that the C+ will most what the current test steps are, so the Contributor can see them and add new / update the existing steps to account for the scenario in the issue, right?

Reply:
accepted!

Reply:
Sorry, I meant, Updated! Should be clearer now, but LMK!